bot/exts/halloween/spooknamerate.py

```diff
@@ -0,0 +1,329 @@
+You can rate each message according to how scary it is
+At the end of the day, the author of the message with most reactions will be the winner of the day.
+On a scale of 1 to {len(self.emojis_val)}, the reactions order:
+{emoji_message}""",
+            inline=False
+        )
+
+        # command to add spookified name
+        help_embed.add_field(
+            name="How do I add my spookified word?",
+            value=f"Just simply type `{self.bot.command_prefix}spooknamerate add my word`",
+            inline=False
+        )
+
+        await ctx.send(embed=help_embed)
+
+    @spook_name_rate.command(name='list', aliases=["all", "entries"])
+    async def list_entries(self, ctx: Context) -> None:
+        """Send all the entries up till now in a single embed."""
+        await ctx.send(embed=await self.get_responses_list(final=False))  # Send a list of users
+
+    @spook_name_rate.command(name='word')
+    async def tell_word(self, ctx: Context) -> None:
+        """Tell the current random word (name)."""
+        if not self.poll:
+            return await ctx.send(f"The name is **{self.word}**")
+        else:
+            return await ctx.send(f"The word ~~is~~ was **{self.word}**. The poll has already started, so you cannot \
+add an entry.")
+
+    @spook_name_rate.command(name="add", aliases=["+", "register"])
+    async def add_word(self, ctx: Context, *, word: str) -> None:
+        """A command that adds your word!"""
+        if not self.poll:
+            message = ctx.message  # get the message
+
+            for message_id in self.messages:
+                data = self.messages.get(message_id)
+                if data['author'] == message.author:  # if the author has already added an entry
+                    return await ctx.send(f"But you have already added an entry! Type `{self.bot.command_prefix}spooknamerate \
+delete` to delete it, and then you can add it again")
+                elif data['word'] == word:
+                    # Someone has added the word
+                    return await ctx.send("TOO LATE. Someone has already added this word.")
+
+            # otherwise
+            self.messages[message.id] = {  # store it
+                'word': word,  # store the word
+                'author': message.author,  # and the author
+                'score': 0,  # and the score
+            }
+
+            for emoji in self.emojis_val.keys():  # get the emojis
+                await message.add_reaction(emoji)  # and add them
+
+            logger.info(f"{message.author} added the word {word!r}")
+            # display if entry was successful
+            return await ctx.send(f"{word!r} added successfully!\n{random.choice(self.added_messages)}")
+
+        else:
+
+            logger.info(f"{ctx.message.author} tried to add a word, but the poll had already started.")
+            await ctx.send("Sorry, the poll has started! You can try and participate in the next round though!")
```

Comment:
I'd rather put this at the top with a `if self.poll`, and fail fast with a return after. That way we removed a whole indenation level of the command logic.

Reply:
Did that, and it looks much better. Thank you!